client/app/bundles/Events/reducers/AttendanceFormReducer.jsx

```diff
@@ -0,0 +1,38 @@
+import {
+  LOOK_UP_MEMBER, SET_ATTENDANCE_ATTRIBUTE, CREATE_ATTENDANCE
+} from '../actions/types';
+
+const INITIAL_STATE = {
+  'primary-email-address': '',
+  'given-name': '',
+  'family-name': '',
+  'primary-phone-number': '',
+  'disabled': ''
+};
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+  case SET_ATTENDANCE_ATTRIBUTE:
+    const { prop, value } = action.payload;
+    return { ...state, [prop]: value };
```

Comment:
{ ...state, [prop]: value };
 Now that's a weird data structure which i haven't seen before.